test/Microsoft.Azure.WebJobs.Extensions.OpenApi.Core.Tests/Visitors/NullableObjectTypeVisitorTests.cs

```diff
@@ -70,6 +70,23 @@ public void Given_Type_When_IsPayloadVisitable_Invoked_Then_It_Should_Return_Res
             result.Should().Be(expected);
         }
 
+        [DataTestMethod]
+        [DataRow(DataType.DateTime, "date-time")]
+        [DataRow(DataType.Date, "date")]
+        public void Given_DataTypeAttribute_When_Visit_Invoked_Then_It_Should_Return_Result(DataType dataType, string expected)
+        {
+            var name = "hello";
+            var acceptor = new OpenApiSchemaAcceptor();
+            var type = new KeyValuePair<string, Type>(name, typeof(DateTime?));
+            var attribute = new DataTypeAttribute(dataType);
+
+            this._visitor.Visit(acceptor, type, this._strategy, attribute);
+
+            acceptor.Schemas.Should().ContainKey(name);
+            acceptor.Schemas[name].Type.Should().Be("string");
```

Comment:
Instead of hard-coding the value `"string"`, can you pass this value through the parameter?